content/docs/nav.yml

```
@@ -1,153 +1,154 @@
+      title: React بدون ES6
     - id: react-without-jsx
       title: React بدون JSX
     - id: reconciliation
-      title: Reconciliation
+      title: المطابقة (Reconciliation)
     - id: refs-and-the-dom
-      title: Refs and the DOM
+      title: refs مع الـ DOM
     - id: render-props
-      title: Render Props
+      title: خاصيات التصيير
     - id: static-type-checking
-      title: Static Type Checking
+      title: التحقّق من الأنواع الثابتة
     - id: strict-mode
-      title: Strict Mode
+      title: الوضع الصارم
     - id: typechecking-with-proptypes
-      title: Typechecking With PropTypes
+      title: التحقق من الأنواع باستخدام PropTypes
     - id: uncontrolled-components
-      title: Uncontrolled Components
+      title: المكونات غير المضبوطة
     - id: web-components
-      title: Web Components
+      title: مكونات الويب
 - title: مراجع API 
   items:
     - id: react-api
-      title: React
+      title: واجهة برمجة التطبيقات (API) ذات المستوى الأعلى في React.
       subitems:
         - id: react-component
-          title: React.Component
+          title: الصنف React.Component
     - id: react-dom
-      title: ReactDOM
+      title: الكائن ReactDOM
     - id: react-dom-server
-      title: ReactDOMServer
+      title: الكائن ReactDOMServer
     - id: dom-elements
-      title: DOM Elements
+      title: عناصر الـ DOM
     - id: events
-      title: SyntheticEvent
+      title: الأحداث المصطنعة (Synthetic Events)
     - id: test-utils
-      title: Test Utilities
+      title: أدوات الاختبار
     - id: shallow-renderer
-      title: Shallow Renderer
+      title: التصيير السطحي
     - id: test-renderer
-      title: Test Renderer
+      title: مصير الاختبار
     - id: javascript-environment-requirements
-      title: JS Environment Requirements
+      title: متطلبات بيئة JavaScript
     - id: glossary
-      title: Glossary
+      title: معجم مصطلحات React 
 - title: Hooks (جديد)
   isOrdered: true
   items:
       - id: hooks-intro
-        title: Introducing Hooks
+        title: مدخل إلى الـ Hooks
       - id: hooks-overview
-        title: Hooks at a Glance
+        title: لمحة خاطفة عن الـ Hooks
       - id: hooks-state
-        title: Using the State Hook
+        title: استعمال Hook الحالة
       - id: hooks-effect
-        title: Using the Effect Hook
+        title: استعمال Hook التأثير
       - id: hooks-rules
-        title: Rules of Hooks
+        title: قواعد استعمال الـ Hooks
       - id: hooks-custom
-        title: Building Your Own Hooks
+        title: بناء Hook خاص بك
       - id: hooks-reference
-        title: Hooks API Reference
+        title: مرجع إلى الواجهة البرمجية للـ Hooks
       - id: hooks-faq
-        title: Hooks FAQ
+        title: الأسئلة الشائعة حول الـ Hooks
 - title: المساهمة
   items:
   - id: how-to-contribute
-    title: How to Contribute
+    title: كيف تساهم
   - id: codebase-overview
-    title: Codebase Overview
+    title: نظرة عامة عن اساس الكود
```

`Code` is often translated as `شفرة برمجية`.

My suggestion is:

```suggestion
    title: نظرة عامة عن قاعدة الشفرة المصدرية
```

But if you can find a better translation that would be perfect :muscle:

---

طبعا ان كانت هناك ترجمة مثالية اكثر لاباس .. فقط هنا قمت بالترجمة ليس الحرفية و انما حسب مضمون  التوثيق ..